src/SignalR/perf/benchmarkapps/Crankier/Readme.md

```diff
@@ -0,0 +1,40 @@
+# Crankier
+
+Load testing for ASP.NET Core SignalR
+
+## Commands
+
+### local
+
+```
+> dotnet run help local
```

Comment:
```suggestion
> dotnet run -- help local
```

While your example works fine, it's probably best to encourage using `--` here. The `--` option tells `dotnet run` that everything after it is for the app being run and not for the `dotnet run` command itself. This is only really relevant when a command/option in the app conflicts with an option in `dotnet run` but it's good to encourage this separation just in case something changes :)